core/reducers/blueprints.js

```diff
@@ -38,6 +39,11 @@ const blueprints = (state = [], action) => {
           }
         ]
       });
+    case FETCHING_BLUEPRINTS:
+      return Object.assign({}, state, {
+        fetchingBlueprints: true,
+        errorState: null
```

Comment:
I don't understand why the FETCHING_BLUEPRINTS reducer should clean up the errorState variable?
Generally, I tend to think of redux actions as single operation components, meaning that they should not affect sometihng extra than what their name says.
In this case, I would expect that the FETCHING_BLUEPRINTS action to be just setting the fetchingBlueprints to true and nothing else. (Even better would be FETCHING_BLUEPRINTS_INIT so that it's clear that this is just initialization action and not actually returning any blueprints list. But since this action existed no need to change that here)

The question about errorState remains though.

Reply:
I refactored the reducers for `blueprints` so that separate reducers control the different things inside the `blueprints` state. I was kinda following this example, where one action can be used by more than one reducer: https://egghead.io/lessons/javascript-redux-displaying-error-messages